content/docs/reference-glossary.md

```diff
@@ -1,43 +1,43 @@
 ---
 id: glossary
-title: Glossary of React Terms
+title: React की शर्तों का शब्दकोष
```

Comment:
terms should be translated as terms in hindi, sharton gives it a different meaning

Reply:
this still has not been resolved?